server/test/buff_server/packages_test.exs

```diff
@@ -1,97 +1,105 @@
 defmodule BuffServer.PackagesTest do
-
-  @s3_bucket_name Application.get_env(:buff_server, :s3_bucket_name)
-  @dummy_artifact File.read("test/buff_server/fixtures/dummy_artifact.gz")
-  @dummy_artifact2 File.read("test/buff_server/fixtures/dummy_artifact2.gz")
   use BuffServer.DataCase, async: true
-  import Mox
-
   alias BuffServer.Packages
+  import Mox
 
   describe "packages" do
     setup :verify_on_exit!
 
-    def artifact_fixture(file) do
-      {:ok, artifact_binary} = file
-      artifact_binary
-    end
-
-    def default_attrs() do
-      %{
-        description: "some description",
-        homepage: "some homepage",
-        keywords: [],
-        name: "some-name",
-        repository_url: "http://aaa.com",
-        artifact_binary: artifact_fixture(@dummy_artifact)
-      }
-    end
+    @s3_bucket_name Application.get_env(:buff_server, :s3_bucket_name)
+    @dummy_artifact File.read("test/buff_server/fixtures/dummy_artifact.gz") |> elem(1)
```

Comment:
elem(1)? that'll take the 2nd element from a Map?

Reply:
i think it takes the first.. lol